prisma/migrations/20251125152154_change_ids_to_uuid/migration.sql

```diff
@@ -0,0 +1,45 @@
+/*
+  Warnings:
+
+  - The primary key for the `User` table will be changed. If it partially fails, the table could be left without primary key constraint.
+
+*/
+-- AlterTable
+ALTER TABLE "User" DROP CONSTRAINT "User_pkey",
+ALTER COLUMN "id" DROP DEFAULT,
+ALTER COLUMN "id" SET DATA TYPE TEXT,
+ADD CONSTRAINT "User_pkey" PRIMARY KEY ("id");
+DROP SEQUENCE "User_id_seq";
```

Comment:
This line drops the sequence "User_id_seq". If the sequence was used by integer PKs, dropping it before migrating data or ensuring no dependent objects exist can cause data loss and database errors.

**Sugerencia:**
```typescript
Do not drop the sequence until after you have fully migrated all dependent data and confirmed no code relies on auto-incrementing integer ids. Prefer to keep the sequence until the migration is complete and the old id column is removed in a controlled step.
```